diff -r -u photivo-20120818.orig/Sources/ptCimg.cpp photivo-20120818/Sources/ptCimg.cpp
--- photivo-20120818.orig/Sources/ptCimg.cpp	2012-08-19 00:35:52.000000000 +0200
+++ photivo-20120818/Sources/ptCimg.cpp	2012-08-27 00:49:49.969047608 +0200
@@ -315,7 +315,7 @@
   }
 
   for (short Threads=0; Threads < NumberOfThreads; Threads++) {
-    CImage[Threads].edge_tensors(Sharpness,Anisotropy,Alpha,Sigma);
+    CImage[Threads].diffusion_tensors(Sharpness,Anisotropy,Alpha,Sigma);
     if (Blur) CImage[Threads].blur(Blur,true);
 
   }
Only in photivo-20120818/Sources: ptCimg.cpp~
