Only in photivo-20120818/: photivo.pro~
diff -r -u photivo-20120818.orig/photivoProject/photivoProject.pro photivo-20120818/photivoProject/photivoProject.pro
--- photivo-20120818.orig/photivoProject/photivoProject.pro	2012-08-19 00:35:53.000000000 +0200
+++ photivo-20120818/photivoProject/photivoProject.pro	2012-08-27 00:24:36.391670616 +0200
@@ -61,6 +61,15 @@
   QMAKE_CC    = ccache /usr/bin/gcc
   QMAKE_CXX   = ccache /usr/bin/g++
 
+  # use a CImg include provided by the system instead of the local copy
+  CONFIG(WithSystemCImg) {
+    PKGCONFIG  += CImg
+    QMAKE_CXXFLAGS_RELEASE     += -DSYSTEM_CIMG
+    QMAKE_CXXFLAGS_DEBUG       += -DSYSTEM_CIMG
+
+    system(echo "Using system supplied CImg library")
+  }
+
   QMAKE_CFLAGS_RELEASE    += -DPREFIX=$${PREFIX} -L$${PREFIX}/lib $$(CFLAGS)
   QMAKE_CXXFLAGS_RELEASE  += -DPREFIX=$${PREFIX} -I$${PREFIX}/include $$(CXXFLAGS)
   QMAKE_CFLAGS_DEBUG      += -DPREFIX=$${PREFIX} -L$${PREFIX}/lib $$(CFLAGS)
Only in photivo-20120818/photivoProject: photivoProject.pro~
diff -r -u photivo-20120818.orig/Sources/ptCimg.cpp photivo-20120818/Sources/ptCimg.cpp
--- photivo-20120818.orig/Sources/ptCimg.cpp	2012-08-19 00:35:52.000000000 +0200
+++ photivo-20120818/Sources/ptCimg.cpp	2012-08-26 23:54:07.582463566 +0200
@@ -77,7 +77,11 @@
   #define cimg_display 0
 #endif
 
-#include "greyc/CImg.h"
+#ifdef SYSTEM_CIMG
+  #include <CImg.h>
+#else
+  #include "greyc/CImg.h"
+#endif
 
 
 using namespace cimg_library;
diff -r -u photivo-20120818.orig/Sources/ptImage_Pyramid.cpp photivo-20120818/Sources/ptImage_Pyramid.cpp
--- photivo-20120818.orig/Sources/ptImage_Pyramid.cpp	2012-08-19 00:35:53.000000000 +0200
+++ photivo-20120818/Sources/ptImage_Pyramid.cpp	2012-08-26 23:53:30.490135414 +0200
@@ -41,8 +41,11 @@
   #define cimg_display 0
 #endif
 
-#include "greyc/CImg.h"
-
+#ifdef SYSTEM_CIMG
+  #include <CImg.h>
+#else
+  #include "greyc/CImg.h"
+#endif
 
 using namespace cimg_library;
 
